scipy/stats/_stats_py.py

```diff
@@ -34,8 +34,12 @@
 import numpy as np
 from numpy import array, asarray, ma
 
+from scipy import sparse
 from scipy.spatial.distance import cdist
+from scipy.spatial import distance_matrix
+
 from scipy.ndimage import _measurements
+from scipy.optimize import milp, LinearConstraint
 from scipy._lib._util import (check_random_state, MapWrapper, _get_nan,
```

Comment:
can you rewrite as

```python
from scipy._lib._util import (
    check_random_state, 
    MapWrapper, 
    _get_nan,
    rng_integers,
    _rename_parameter,
    _contains_nan,
    AxisError,
)
```

sorry it has nothing to do with your PR but it is really difficult to read :-)

Reply:
Hi @VascoSch92, we generally avoid touching lines which are unrelated to the PR. For style-only changes like this, we prefer to deal with them only when the lines are already being touched, or in wholesale changes. This makes reviewing individual PRs easier and keeps the git blame history more clean. See https://scipy.github.io/devdocs/dev/contributor/pep8.html#pep8-and-scipy.

Reply:
ah ok ;-) 

sorry i was just reading the PR and i saw that.